src/run/mod.rs

```diff
@@ -33,3 +33,9 @@ pub use self::run::{ComparisonsIter, Run};
 pub use self::segment_history::SegmentHistory;
 pub use self::segment::Segment;
 pub use self::editor::Editor;
+
+/// Checks a given name against the current comparisons in the Run to
+/// ensure that it is valid for use.
+pub fn validate_comparison_name(run: &Run, new: &str) -> bool {
+    !new.starts_with("[Race]") && !run.comparisons().any(|c| c == new)
+}
```

Comment:
I'm not sure if it should be here. Maybe it should be in livesplit_core::comparison? Or maybe as a method on Run?

Reply:
I was thinking of putting I only put it in the `src/run/mod.rs` because the only two uses that I saw were children of Run (editor and Run itself) so I put it here. Would make sense to move it to comparison though.